content/episode/devney-hamilton-sassafras-tech.md

```diff
@@ -0,0 +1,351 @@
++++
+title = "Devney Hamilton on Sassafras Tech Collective"
+date = 2021-03-18T15:08:22-07:00
+description = "Devney Hamilton talks about Sassfras Tech Collective, Sassafras' internal structures, and the importance of prefigurative processes."
+tags = []
+draft = false
+podcast_file = "https://ia601506.us.archive.org/22/items/devney-hamilton-on-sassafras-tech-collective/Devney%20Hamilton%20on%20Sassafras%20Tech%20Collective.mp3"
+podcast_duration = "47:44"
+podcast_bytes = "45933627"
+episode_image = "/img/devney-hamilton-sassfras.jpg"
++++
+
+## Episode Notes
+
+Sassafras encourages people of color and/or with disabilities and/or trans or gender-non-conforming, or otherwise targeted with oppression in tech to contact them at [info@sassafras.coop](mailto:info@sassafras.coop).
+
+Ways to stay in touch with Sassafras:
+* Website: [sassafras.coop](https://www.sassafras.coop/)
+* Twitter: [@sassafrastech](https://twitter.com/sassafrastech)
+* Blog: [blog.sassafras.coop](https://blog.sassafras.coop/)
```

Comment:
Thanks! For now could we please swap the positions of the Website and Blog? The Blog is most up-to-date for the foreseeable future :)

Reply:
Done.